WebRoot/templates/share/Dialogs.template

```diff
@@ -725,9 +725,7 @@
 <template id='share.Dialogs#ZmTwoFactorCustomLoginPage'>
 	<div class="ZmTwoFactorCustomSetupLoginPage">		
 		<div class="logo">
-			<a href="https://www.zimbra.com/"  style=" outline :none;" id="bannerLink" target="_new" title='<fmt:message key="zimbraTitle"/>'>
-				<img style="outline:none;" src="/img/new-logo.png" width="130" height="32" border="0" alt="Zimbra" title="Zimbra">
-			</a>
+			<img style="outline:none;" src="/img/new-logo.png" width="130" height="32" border="0" alt="Zimbra" title="Zimbra">
```

Comment:
why have we removed links to zimbra.com ?

Reply:
earlier we dint have , urls here and we only had logo , so had talk with shardool and he suggested to remove it from both